modules/polling/api/fetchAllCurrentVotes.ts

```diff
@@ -1,27 +1,30 @@
 import { gqlRequest } from 'modules/gql/gqlRequest';
 import { allCurrentVotes } from 'modules/gql/queries/allCurrentVotes';
+import { AllCurrentVotesQueryVariables } from 'modules/gql/types';
 import { SupportedNetworks } from 'modules/web3/constants/networks';
 import { networkNameToChainId } from 'modules/web3/helpers/chain';
 import { parseRawOptionId } from '../helpers/parseRawOptionId';
 import { PollTallyVote } from '../types';
 
 export async function fetchAllCurrentVotes(
   address: string,
-  network: SupportedNetworks
+  network: SupportedNetworks,
+  variables: AllCurrentVotesQueryVariables
 ): Promise<PollTallyVote[]> {
   const data = await gqlRequest({
     chainId: networkNameToChainId(network),
     query: allCurrentVotes,
-    variables: { argAddress: address.toLowerCase() }
+    variables
```

Comment:
Now that the query signature has been updated we can pass any of the variables we want to the query